src/loan_form/LoanForm.jsx

```diff
@@ -4,21 +4,59 @@ import LoanEntries from './LoanEntries'
 import AddLoan from './buttons/AddLoanButton'
 import Calculate from './buttons/CalculateButton'
 import emptyLoan from '../models/loan/emptyLoan'
-
-const calculate = e => console.log('calculate')
+import Calculation from './loan_calculate/Calculation'
 
 export default class LoanForm extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-      loans: [emptyLoan(0)]
+      loans: [emptyLoan(0)],
+      maxTotalMonths: 0
     }
+    this.handleChange = this.handleChange.bind(this)
```

Comment:
FYI `@autobind` does this for you :)

Reply:
that's what I get for going boilerplate. I'll remove this